src/Event/ValueObjects/SubEventUpdate.php

```diff
@@ -47,4 +48,26 @@ public function getBookingAvailability(): ?BookingAvailability
     {
         return $this->bookingAvailability;
     }
+
+    public static function fromUdb3ModelsSubEventUpdate(
+        Udb3ModelsSubEventUpdate $udb3ModelsSubEventUpdate
+    ): self {
+        $update = new self($udb3ModelsSubEventUpdate->getSubEventId());
+
+        $udb3ModelsStatus = $udb3ModelsSubEventUpdate->getStatus();
```

Comment:
Because this code is inside the `SubEventUpdate` class itself, this shorthand would also be valid
```
$update->status = $udb3ModelsSubEventUpdate->getStatus() ? Status::fromUdb3ModelStatus($udb3ModelsSubEventUpdate->getStatus()) : null;
```
But I am perfectly happy with the `with` methods 😊

Reply:
This class does not exist anymore after your other suggestion to remove it, so "done" 😉